sdk/tracing/azure-core-tracing-opencensus/pom.xml

```diff
@@ -72,5 +77,14 @@
       <version>5.0.0-preview.4</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
```

Comment:
Shouldn't have this as a non test dependency right?

Reply:
It is a test dependency.

Reply:
Didn't need this dependency, removed it.